### ISSUE TITLE
[bug] title and caption not set when using gemini-flash-2.0

### ISSUE THREAD

**Issue by hensing** [bmachek/lrc-ai-assistant#19]
**Summary:**

Hi there! I've noticed that the Title, Caption, and Alt Text fields aren't automatically populated with the generated descriptions when using the Gemini-Flash-2.0 engine. This functionality seems to work perfectly fine with Gemini-Flash-1.5.  I suspect there might be an issue with the 2.0 engine implementation.

**Steps to Reproduce:**

1.  Open Lightroom Classic and set plugin to gemini-flash-2.0 
2.  Annotate an image as usual
3.  Observe that the Title, Caption, and Alt Text fields remain empty. They keywords are populated.

**Expected Result:**

The Title, Caption, and Alt Text fields should be automatically populated with the generated image descriptions and keywords

**Actual Result:**

The Title, Caption, and Alt Text fields remain empty.

**Environment:**

*   Lightroom Classic Version: 14.2 classic
*   Plugin Version: both: 2.0.0 and latest commit (5a4d5dd)
*   Operating System: Win 11
*   Engine: Gemini-Flash-2.0

**Additional Notes:**
Just to confirm, the auto-population of these fields works as expected when I switch back to the Gemini-Flash-1.5 engine.

**Comment by bmachek** [comment]
Unfortunately, I can't do anything about it at the moment. The code in the plugin for all Gemini models is the same, so the issue is with Google. I recommend using gpt4o-mini or gemini1.5-pro, these deliver the best results. The Google Gemini flash models tend to deliver faulty answers also in a technical way (like repeating the same keyword 100 times and more). And I don't want to handle these errors in the plugin, as it makes no sense.
